src/components/Payment.tsx

```
@@ -42,48 +25,18 @@
   );
 
   useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: isValid

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)